types/stripe-v3/index.d.ts

```diff
@@ -43,27 +43,47 @@ declare namespace stripe {
         retrievePaymentIntent(
             clientSecret: string,
         ): Promise<PaymentIntentResponse>;
+        // Deprecated:
```

Comment:
I suggest using [`@deprecated`](https://jsdoc.app/tags-deprecated.html):
```suggestion
        /** @deprecated */
```

![image](https://user-images.githubusercontent.com/10235949/70291253-46bc9300-17e3-11ea-853c-9e3fa231417b.png)